lib/Database/lmdb/mdb.c

```diff
@@ -9790,7 +9790,7 @@ mdb_page_split(MDB_cursor *mc, MDB_val *newkey, MDB_val *newdata, pgno_t newpgno
 			 * the split so the new page is emptier than the old page.
 			 * This yields better packing during sequential inserts.
 			 */
-			if (nkeys < 32 || nsize > pmax/16 || newindx >= nkeys) {
+			if (nkeys < env->me_os_psize/128 || nsize > pmax/16 || newindx >= nkeys) {
```

Comment:
I would suggest adding a comment here identifying that there is an IndexStoreDB-specific change and why. 

Should this be `me_psize` instead of `me_os_psize`? They're usually the same, but it's probably more correct to use the actual db page size and not the os page size.

Reply:
Will do. And yes, it probably should be. I actually have that change locally now 😅. Was just looking whether I could add a test for this case or not.